akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -91,10 +95,21 @@ private[http2] class Http2ServerDemux(http2Settings: Http2CommonSettings, initia
       override def settings: Http2CommonSettings = http2Settings
       override def isUpgraded: Boolean = upgraded
 
+      def maxConcurrentStreams: Int = http2Settings.maxConcurrentStreams
+
       override protected def logSource: Class[_] = if (isServer) classOf[Http2ServerDemux] else classOf[Http2ClientDemux]
 
       val multiplexer = createMultiplexer(frameOut, StreamPrioritizer.first())
 
+      // TODO: we only support the initial SETTINGS message (the only one that's compulsory)
+      //  so we're keeping the settings as a field instead of implementing SETTINGS-SETTINGS_ACK
+      //  correlation logic.
+      // TODO: the receiver of a SETTINGS frame must apply them in the order they
+      //  are received. Review order of the settings in the sequence since it's relevant.
+      val localSettings = immutable.Seq(
```

Comment:
Let's make it a `def` and call it `initialSettings`.

Reply:
I used `localSettings` so it's clear it's not settings received from the peer. How about `initialLocalSettings`?

Reply:
Sounds good